pymc/distributions/discrete.py

```diff
@@ -462,15 +462,16 @@ def rng_fn(cls, rng, q, beta, size):
 
 
 class DiscreteWeibull(Discrete):
-    R"""Discrete Weibull log-likelihood
+    """Discrete Weibull log-likelihood.
 
     The discrete Weibull distribution is a flexible model of count data that
     can handle both over- and under-dispersion.
     The pmf of this distribution is
 
-    .. math:: f(x \mid q, \beta) = q^{x^{\beta}} - q^{(x + 1)^{\beta}}
+    .. math:: f(x \\mid q, \beta) = q^{x^{\beta}} - q^{(x + 1)^{\beta}}
```

Comment:
I think this extra `\` will also break latex rendering. The formulas already look right in `latest`: https://www.pymc.io/projects/docs/en/latest/api/distributions/generated/pymc.DiscreteWeibull.html so they should not be modified.

Reply:
same for the other extra `\` characters in math formulas

Reply:
Strange, as the extra \ doesn't appear in my local copy...

Reply:
maybe your ide added that automatically when you removed the R? I think double backlashes are escaped in regular strings, but the previed on readthedocs ended up having latex errors

Reply:
Yeah, it was the missing R. Thanks